### ISSUE TITLE
UniDriver interface should support `debug` method

### ISSUE THREAD

**Issue by argshook** [wix-incubator/unidriver#6]
hello!

i may be spoiled by enzymes' `debug` (which returns HTML representation of component tree) but it seems such method would be really helpful. just for those cases when you simply want to see some shape of what you're working with without getting too deep into details.

perhaps something like this is already planned?

**Comment by GabiGrin** [comment]
hey @argshook 
this sounds like a good addition, but not sure adding it to the small api surface of UniDriver 
Maybe it can be an a outside method that does it? something like:

import {debug} from 'unidriver'

```
it('does something of great value', async () => {
  // test test test
  console.log('wth is going on', await debug(driver));
  // more test
});
```

WDYT?
@borisd9 thoughts?

**Comment by argshook** [comment]
yeah `import {debug} from 'unidriver'` can work just as well imho

**Comment by borisd9** [comment]
@GabiGrin Yes, i was hesitant on adding `debug` on every adapter as well.
separate `debug` would be nicer, and optional to be imported, and would be aware of the driver's context in order to print its HTML
